packages/card-react/src/TaskCard.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from "react";
+import cn from "classnames";
+import { PaddingOptions } from "./types";
+
+export interface TaskCardProps extends PaddingOptions {
+    /**
+     * Bakgrunnsfargen til kortet
+     * @default "hvit"
+     */
+    bgColor?: "hvit" | "snohvit" | "sand" | "dis";
+    /**
+     * Skal kortet ha skygge?
+     * @default false
+     */
+    shadow?: boolean;
+    className?: string;
+}
+
+export const TaskCard: FC<TaskCardProps> = ({
+    bgColor = "hvit",
+    shadow = false,
+    padding = "l",
+    topPadding,
+    children,
+    className,
+}) => {
+    const spacingClass = `jkl-spacing-${padding}--all`;
+    const topSpacingClass = topPadding ? `jkl-spacing-${topPadding}--top` : "";
+    return (
+        <div
+            className={cn("jkl-task-card", className, {
+                [`jkl-task-card--${bgColor}`]: bgColor,
+                "jkl-task-card--shadow": shadow,
+            })}
```

Comment:
Siden vi gir en default er `bgColor` alltid truthy. Med forbehold om at Prettier kanskje er uenig i formateringen:

```suggestion
            className={cn("jkl-task-card", `jkl-task-card--${bgColor}`, className, {
                "jkl-task-card--shadow": shadow,
            })}
```